atd-vze/src/Components/GridTableWidgets.js

```diff
@@ -23,10 +23,13 @@ const GridTableWidgets = ({ aggData, widgetsConfig }) => {
   };
 
   const formatWidgetData = (data, format) => {
-    if (format === "dollars") {
+    // Handles null return values and format
+    if (!!data && format === "dollars") {
       return formatCostToDollars(data);
-    } else {
+    } else if (!!data) {
       return data.toString();
+    } else {
+      return "0";
```

Comment:
just wondering if "0" is always going to be the right value to return. Maybe "--" or something that indicates "no data" might be a better one-size-fits-all solution?

Reply:
This is a good idea because "0" could be misleading. I'll add "--" for now and we could update with something else if we think of a good icon or if it produces any confusion to test users.